.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
No, if we call it ``TRAVIS_CACHE_DIR``, then its default name should not be ``.mink-debug-cache``, agree? :)